pkg/validation/config.go

```diff
@@ -402,6 +407,21 @@ func validatePromotionConfiguration(fieldRoot string, input api.PromotionConfigu
 	if len(input.Name) != 0 && len(input.Tag) != 0 {
 		validationErrors = append(validationErrors, fmt.Errorf("%s: both name and tag defined", fieldRoot))
 	}
+
+	if promotesOfficialImages {
+		var latestRelease bool
+		if releaseTagConfiguration != nil {
+			latestRelease = true
+		}
+		if releases != nil {
+			if _, ok := releases["latest"]; ok {
+				latestRelease = true
+			}
+		}
+		if !latestRelease {
+			validationErrors = append(validationErrors, fmt.Errorf("promotion to the integration image stream or namespace requires defining tag_specification or releases[\"latest\"]"))
```

Comment:
This is also too implementation-specific for a user message.

Importing a release is just a proxy, the validation error should describe the _purpose_ of the verification and what we expect from the configuration, not the mechanics of what was checked.

Reply:
At least my purpose is to ask our users to add the `tag_specification` or `releases`.
Are you suggesting our purpose is to add an e2e tests instead?